src/BlazorUI/Bit.BlazorUI.Tests/SnackBar/BitSnackBarTests.cs

```diff
@@ -0,0 +1,205 @@
+        DataRow("title", "")
+    ]
+    [TestMethod]
+    public async Task BitSnackBarShowTest(string title, string body)
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        await com.Instance.Show(title, body);
+
+        var item = com.Find(".bit-snb-itm");
+        Assert.IsNotNull(item);
+
+        var titleElement = com.Find(".bit-snb-ttl");
+        Assert.AreEqual(title, titleElement.InnerHtml);
+
+        if (body.IsNullOrEmpty() is false)
+        {
+            var bodyElement = com.Find(".bit-snb-bdy");
+            Assert.AreEqual(body, bodyElement.InnerHtml);
+        }
+    }
+
+    [DataTestMethod,
+        DataRow(true),
+        DataRow(false)
+    ]
+    [TestMethod]
+    public async Task BitSnackBarAutoDismissTest(bool autoDismiss)
+    {
+        var com = RenderComponent<BitSnackBar>(
+            parameters =>
+            {
+                parameters.Add(p => p.AutoDismiss, autoDismiss);
+            }
+        );
+
+        await com.Instance.Show("title");
+
+        //Added a sec delay to be sure we are asserting after AutoDismissTime passed
+        await Task.Delay(com.Instance.AutoDismissTime + TimeSpan.FromSeconds(1));
+
+        var items = com.FindAll(".bit-snb-itm");
+
+        Assert.AreEqual(autoDismiss ? 0 : 1, items.Count);
+    }
+
+    [DataTestMethod,
+        DataRow(BitSnackBarType.Info),
+        DataRow(BitSnackBarType.Warning),
+        DataRow(BitSnackBarType.Success),
+        DataRow(BitSnackBarType.Error),
+        DataRow(BitSnackBarType.SevereWarning),
+        DataRow(null)
+    ]
+    [TestMethod]
+    public async Task BitSnackBarTypeTest(BitSnackBarType? type)
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        if (type.HasValue)
+            await com.Instance.Show("title", string.Empty, type);
+        else
+            await com.Instance.Show("title");
+
+        var element = com.Find(".bit-snb-itm");
+
+        var typeClass = type switch
+        {
+            BitSnackBarType.Info => $"bit-snb-info",
+            BitSnackBarType.Warning => $"bit-snb-warning",
+            BitSnackBarType.Success => $"bit-snb-success",
+            BitSnackBarType.Error => $"bit-snb-error",
+            BitSnackBarType.SevereWarning => $"bit-snb-severe-warning",
+            _ => "bit-snb-info"
+        };
+
+        Assert.IsTrue(element.ClassList.Contains(typeClass));
+    }
+
+    [TestMethod]
+    public async Task BitSnackBarCloseButtonTest()
+    {
+        var com = RenderComponent<BitSnackBar>();
+
+        await com.Instance.Show("title");
+
+        var closeButton = com.Find(".bit-snb-cbt");
+        closeButton.Click();
+
+        var items = com.FindAll(".bit-snb-itm");
+
+        Assert.AreEqual(0, items.Count);
+    }
+
+    [TestMethod]
+    public async Task BitSnackBarDismissIconNameTest()
+    {
+        var com = RenderComponent<BitSnackBar>(
+            parameters =>
+            {
+                parameters.Add(x => x.DismissIconName, "Go");
```

Comment:
about this string `Go` here:
always introduce these expected values as method arguments (test data), and plz do not use repeating string values like this.

Reply:
Of course, my bad